lib/coverband/adapters/base.rb

```diff
@@ -64,24 +69,34 @@ def covered_files
 
       def split_coverage(types)
         types.reduce({}) do |data, type|
-          data.update(type => coverage(type))
+          if type == Coverband::RUNTIME_TYPE && Coverband.configuration.simulate_oneshot_lines_coverage
+            data.update(type => simulated_runtime_coverage)
+          else
+            data.update(type => coverage(type))
+          end
         end
       end
 
+      def simulated_runtime_coverage
+        runtime_data = coverage(Coverband::RUNTIME_TYPE)
+        eager_data = coverage(Coverband::EAGER_TYPE)
+        eager_data.each_pair do |_key, vals|
+          vals['data'].map! { |el| el ? (0 - el) : el }
```

Comment:
Looks like el is the coverage for a line in a file. I think a more descriptive variable might be helpful when reading this code. Something like this?

```ruby
vals['data'].map! { |line_coverage| line_coverage ? (0 - line_coverage) : line_coverage }
```

Reply:
done